# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/tabs/src/sync/full_sync.rs
# Commit: 2147c5c81edf
# Full Hash: 2147c5c81edf9cd56c19e7e92dfebfcd77329978
# Author: Ryan VanderMeulen <ryanvm@gmail.com>
# Date: 2023-03-01 17:52 -0500
# Regressor Bug: 1818349
# File Overlap Count: 24
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Backed out changesets b05343a5b533 and 57b2cda7e0db (bug 1818349) for causing bug 1819723.
# ==============================================================================

diff -r d59b76766f0d -r 2147c5c81edf third_party/rust/tabs/src/sync/full_sync.rs
--- a/third_party/rust/tabs/src/sync/full_sync.rs	Wed Mar 01 05:53:30 2023 +0000
+++ b/third_party/rust/tabs/src/sync/full_sync.rs	Wed Mar 01 17:52:01 2023 -0500
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use crate::{sync::engine::TabsSyncImpl, ApiResult, TabsEngine, TabsStore};
+use crate::{sync::engine::TabsSyncImpl, ApiResult, Result, TabsEngine, TabsStore};
 use error_support::handle_error;
 use interrupt_support::NeverInterrupts;
 use std::sync::Arc;
@@ -11,7 +11,7 @@
 use sync15::KeyBundle;
 
 impl TabsStore {
-    #[handle_error(crate::Error)]
+    #[handle_error]
     pub fn reset(self: Arc<Self>) -> ApiResult<()> {
         let mut sync_impl = TabsSyncImpl::new(Arc::clone(&self));
         sync_impl.reset(&EngineSyncAssociation::Disconnected)?;
@@ -19,7 +19,7 @@
     }
 
     /// A convenience wrapper around sync_multiple.
-    #[handle_error(crate::Error)]
+    #[handle_error]
     pub fn sync(
         self: Arc<Self>,
         key_id: String,
